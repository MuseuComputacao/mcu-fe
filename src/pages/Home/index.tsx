import React from 'react';
import { View } from 'react-native';

const Home = (props: any) => {
    return (
      <View style={{ width: '100%', height: '100vh'}}>
        <iframe src='https://museucomputacao.github.io' width={'100%'} height={'100%'} frameBorder={0} />
      </View>
    );
  }

export default Home;