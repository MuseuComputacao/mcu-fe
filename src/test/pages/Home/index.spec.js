import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components'
import Home from '../../../pages/Home/index';

describe('<Home />', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Home />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});