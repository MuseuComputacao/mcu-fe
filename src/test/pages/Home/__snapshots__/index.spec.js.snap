// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> renders correctly 1`] = `
<View
  style={
    Object {
      "height": "100vh",
      "width": "100%",
    }
  }
>
  <iframe
    frameBorder={0}
    height="100%"
    src="https://museucomputacao.github.io"
    width="100%"
  />
</View>
`;
